godtools/App/Features/Dashboard/Presentation/Tools/AllToolsContentView.swift

```diff
@@ -39,6 +39,10 @@ struct AllToolsContentView: View {
                         
                         AllToolsList(viewModel: viewModel, width: width, leadingTrailingPadding: leadingTrailingPadding)
                         
+                        Rectangle()
+                            .frame(height: 80)
```

Comment:
Hey @rachaelblue this would work if the Dashboard TabBar height was fixed to 80 and that value was stored in a constant that could be used in these 3 places.  

I would prefer something more adaptive here instead of a fixed spacing.  

That could be achieved by having a containing view in the Dashboard where the bottom of the container touches the top of the TabBar.  Then any views in the dashboard would be placed inside of the container. 

![dashboard-view](https://user-images.githubusercontent.com/59846460/214324267-273244a7-67bc-49c7-8e8b-d2d231164e51.png)

Reply:
@levieggertcru ok so I changed the `ZStack` that held the tabs and tab bar to a `VStack`, which solved in the initial problem.  But in doing so, I discovered that the view modifier to hide the system tab bar wasn't working like I had expected, and on iOS 16 it was now visible above the custom tab bar when a tab appears for the 2nd time.  I had trouble finding much information about how it should be used... it seems like people either use the system tab bar, or they create a custom one completely from scratch (not the hybrid method we currently have).  So I guess that's an option moving forward if we continue to have difficulties, but for now doing `UITabBar.appearance().isHidden = true` works properly on all iOS versions.

Reply:
Thanks @rachaelblue I'll take a look!  Yeah sometimes I prefer to create from scratch, especially if I'm fighting the system.  It's sometimes easier to just build those components from scratch and this way you know it's going to work correctly across iOS versions.